### PR TITLE
Fix: event.origin and event.environment tags have wrong value for iOS

### PR DESCRIPTION
**Pull request by marandaneto**
## :scroll: Description
Fix: event.origin and event.environment tags have wrong value for iOS

## :bulb: Motivation and Context
Bug found by @jennmueng while writing integrations for Cordova

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

File: flutter/ios/Classes/SwiftSentryFlutterPlugin.swift [hunk 1]
```diff
@@ -201,16 +201,14 @@ public class SwiftSentryFlutterPlugin: NSObject, FlutterPlugin {
             case "sentry.dart.flutter":
                 setEventEnvironmentTag(event: event, origin: "flutter", environment: "dart")
             case "sentry.cocoa":
-                setEventEnvironmentTag(event: event, origin: "flutter", environment: "dart")
-            case "sentry.native":
-                setEventEnvironmentTag(event: event, origin: "flutter", environment: "dart")
+                setEventEnvironmentTag(event: event, origin: "ios", environment: "native")
```
Comment (marandaneto) [inline flutter/ios/Classes/SwiftSentryFlutterPlugin.swift:204]: `sentry.native` only makes sense for Android

Comment (marandaneto) [inline flutter/ios/Classes/SwiftSentryFlutterPlugin.swift:204]: `native` means ObjC/Swift/C/C++ or whatever supported language on iOS, we don't know the language unless there's a way I'm not aware of, maybe @philipphofmann ?

on Android we know when an event was captured on "Java" or "C"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+* Fix: event.origin and event.environment tags have wrong value for iOS
```
Comment (marandaneto) [inline CHANGELOG.md]:
```suggestion
* Fix: event.origin and event.environment tags have wrong value for iOS (#365)
```

**Review by jennmueng — APPROVED** [review]

**Review by ueman — APPROVED** [review]
LGTM

**Pull request merged**